content/en/news/releases/1.22.x/announcing-1.22/change-notes/index.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Istio 1.22.0 Change Notes
+linktitle: 1.22.0
+subtitle: Minor Release
+description: Istio 1.22.0 release notes.
+publishdate: 2024-05-13
+release: 1.22.0
+weight: 10
+aliases:
+    - /news/announcing-1.22.0
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definition). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** usage of `values.istio_cni` in favor of `values.pilot.cni`
+  ([Issue #49290](https://github.com/istio/istio/issues/49290))
+
+## Traffic Management
+
+- **Improved** `ServiceEntry` with `resolution: NONE` to respect `targetPort`, if specified.
+This is particularly useful when doing TLS origination, allowing to set `port:80, targetPort: 443`.
+If undesired set `--compatibilityVersion=1.21` to revert to the old behavior, or remove the `targetPort` specification.
+
+- **Improved** XDS generation to send less resource when possible, sometimes omitting a response entirely.
+This can be disabled by the `PILOT_PARTIAL_FULL_PUSHES=false` environment variable, if necessary.
+  ([Issue #37989](https://github.com/istio/istio/issues/37989)),([Issue #37974](https://github.com/istio/istio/issues/37974))
+
+- **Added** support for skipping the initial installation of CNI entirely.
+
+- **Added** a node taint controller to istiod which removes the `cni.istio.io/not-ready` taint from a node once the Istio CNI pod is ready on that node.
+  ([Issue #48818](https://github.com/istio/istio/issues/48818)),([Issue #48286](https://github.com/istio/istio/issues/48286))
+
+- **Added** endpoints acked generation to the proxy distribution report available through the pilot debug API `/debug/config_distribution`.
+  ([Issue #48985](https://github.com/istio/istio/issues/48985))
+
+- **Added** support for configuring waypoint in Service.
+
+- **Added** capability to annotate pods, services, namespaces and other similar kinds with an annotation, `istio.io/use-waypoint`, to specify a waypoint in the form `[<namespace name>]/<waypoint name>`. This replaces the old requirement for waypoints either being scoped to the entire namespace or to a single service account. Opting out of a waypoint can also be done with a value of `#none` to allow a namespace-wide waypoint where specific pods or services are not guarded by a waypoint allowing greater flexibility in waypoint specification and use.
+  ([Issue #49436](https://github.com/istio/istio/issues/49436))
+
+- **Added** Implement support in waypoint for istio.io/waypoint-for annotation.  ([Issue #49851](https://github.com/istio/istio/issues/49851))
+
+- **Added** a check to prevent creation of ztunnel config when user has specified a gateway as `targetRef` in their AuthorizationPolicy
```

Comment:
```suggestion
- **Added** a check to prevent creation of ztunnel config when user has specified a gateway as `targetRef` in their AuthorizationPolicy.
```